docs/Timers/index.md

```diff
@@ -0,0 +1,71 @@
+# Timers
+
+## Instance Timers
+
+Memory management is handled all for you. To create a timer you run `CF_Timer.Create` with the arguments you specify. Note that an interval of `0.0` means the timer will tick every frame.
```

Comment:
Yes, I do mean every frame. For example, moving a panel across the screen. It would look and feel smoother when done per frame at an unlocked frame rate.